pkg/apis/networking/register.go

```diff
@@ -42,4 +42,17 @@ const (
 	// SKSLabelKey is the label key that SKS Controller attaches to the
 	// underlying resources it controls.
 	SKSLabelKey = GroupName + "/serverlessservice"
+
+	// ServiceTypeKey is the label key attached to a service specifying the type of service.
+	// e.g. Public, Metrics
+	ServiceTypeKey = GroupName + "/serviceType"
+)
+
+const (
+	// Label value for Private services
```

Comment:
Private and public are both for user app, but one is where user requests are routed, second is for internal bookkeeping.